src/installing-workflow/quickstart.md

```diff
@@ -14,7 +14,7 @@ Choose one of the following providers and deploy a new Kubernetes cluster:
 - [Google Container Engine](https://cloud.google.com/container-engine/docs/before-you-begin)
 - [Vagrant](http://kubernetes.io/v1.1/docs/getting-started-guides/vagrant.html)
 
-Reference [this table](http://kubernetes.io/v1.1/docs/getting-started-guides/#table-of-solutions) in the official Kubernetes documentation for a more extensive (but still non-exhaustive) list of Kubernetes provisioning options supported by the project or the community.
```

Comment:
I actually somewhat dislike removing the "non-exhaustive" bit.  Team members have their own side projects like kube-solo or micro-kube that have gained traction within our own team, but are not on the official list.  kube-aws, which we use a lot, I think is in the same boat.  I think it's good for our readers/users to understand that there are myriad viable options for going zero to k8s.